public/js/components/Scopes.js

```diff
@@ -39,7 +39,7 @@ function getSpecialVariables(pauseInfo, parentName) {
     vars.push({
       name: "<exception>",
       path: parentName + "/<exception>",
-      contents: { value: thrown.toJS() }
+      contents: { value: thrown }
```

Comment:
oh sorry, was giving you steps so that you could run it. I pinged you separately on irc to see what you were doing. But yea, thats definitely the issue. I'll make sure both cases are checked.